plugins/shell/src/commands.rs

```diff
@@ -302,6 +303,20 @@ pub fn kill<R: Runtime>(
     Ok(())
 }
 
+#[tauri::command]
+pub fn wait<R: Runtime>(
```

Comment:
It will block the whole app to wait for the process

Reply:
I use async to avoid blocking thread